Makefile

```diff
@@ -97,16 +134,36 @@ lima-and-qemu: networks.yaml
 	rm -rf $(OUTDIR)/downloads
 
 
+FINCH_IMAGE_LOCATION ?=
+FINCH_IMAGE_DIGEST ?=
+ifeq ($(GOOS),windows)
+    # Because the path in windows /C:/<some-path> is not an Absolute path, prefix with file:/ which is handled by lima https://github.com/lima-vm/lima/blob/da1260dc87fb30345c3ee7bfb131c29646e26d10/pkg/downloader/downloader.go#L266
+	FINCH_IMAGE_LOCATION := "file:/$(FINCH_ROOTFS_LOCATION)"
+	FINCH_IMAGE_DIGEST := $(FINCH_ROOTFS_DIGEST)
+else
+	FINCH_IMAGE_LOCATION := $(FINCH_OS_IMAGE_LOCATION)
+	FINCH_IMAGE_DIGEST := $(FINCH_OS_DIGEST)
+endif
 .PHONY: finch.yaml
 finch.yaml: finch-core
 	mkdir -p $(OUTDIR)/os
 	cp finch.yaml $(OUTDIR)/os
 	# using -i.bak is very intentional, it allows the following commands to succeed for both GNU / BSD sed
 	# this sed command uses the alternative separator of "|" because the image location uses "/"
-	sed -i.bak -e "s|<finch_image_location>|$(FINCH_OS_IMAGE_LOCATION)|g" $(OUTDIR)/os/finch.yaml
+	sed -i.bak -e "s|<finch_image_location>|$(FINCH_IMAGE_LOCATION)|g" $(OUTDIR)/os/finch.yaml
+	sed -i.bak -e "s/<finch_image_arch>/$(LIMA_ARCH)/g" $(OUTDIR)/os/finch.yaml
+	sed -i.bak -e "s/<finch_image_digest>/$(FINCH_IMAGE_DIGEST)/g" $(OUTDIR)/os/finch.yaml
```

Comment:
maybe we can consolidate `finch.yaml` and `finch.windows.yaml` rules to be something like:

```
WINDOWS ?=
ifeq ($(GOOS),windows)
    WINDOWS=.windows
endif

finch$(WINDOWS).yaml: finch-core
	mkdir -p $(OUTDIR)/os
	cp finch.yaml $(OUTDIR)/os
	# using -i.bak is very intentional, it allows the following commands to succeed for both GNU / BSD sed
	# this sed command uses the alternative separator of "|" because the image location uses "/"
	sed -i.bak -e "s|<finch_image_location>|$(FINCH_IMAGE_LOCATION)|g" $(OUTDIR)/os/finch.yaml
	sed -i.bak -e "s/<finch_image_arch>/$(LIMA_ARCH)/g" $(OUTDIR)/os/finch.yaml
	sed -i.bak -e "s/<finch_image_digest>/$(FINCH_IMAGE_DIGEST)/g" $(OUTDIR)/os/finch.yaml
```